Library/Formula/logstash.rb

```diff
@@ -2,19 +2,20 @@
 
 class Logstash < Formula
   homepage 'http://logstash.net/'
-  url 'https://download.elasticsearch.org/logstash/logstash/logstash-1.4.2.tar.gz'
-  sha1 'd59ef579c7614c5df9bd69cfdce20ed371f728ff'
+  url 'https://download.elasticsearch.org/logstash/logstash/logstash-1.5.0.tar.gz'
+  sha1 '9729c2d31fddaabdd3d8e94c34a6d1f61d57f94a'
```

Comment:
If you could bump this checksum to `sha256` when you fix the homepage as well, that'd be :tada: :fireworks:.